B-Minus-Programmers/jojibrush
# Canvas class/Rendering Area

Wrote the Canvas class - which maintains a vector of all of the shapes on the image, and calls draw() on each shape, providing a QPainter pointed to the Canvas widget.

**To Test:**
Checkout the `render-test` branch, which contains a basic shape. The project should build and render a diagonal line in the top-left corner. 

Please also test that the program icon is Joji's face.

Closes #3
